src/modules/Home/components/AccountBalances/AccountTotalBalances.tsx

```diff
@@ -1,20 +1,27 @@
 import { BoxProps } from '@chakra-ui/react';
 
 import TotalBalance from '@/components/Common/TotalBalance';
-import { useGetCoingeckoCoin } from '@/hooks/queries/useGetCoingeckoCoin';
+import { usePrice } from '@/hooks/queries/usePrice';
 
 export type AccountBalancesProps = BoxProps;
 
 const AccountTotalBalances = () => {
   const {
-    data: { marketCap, dailyVolume } = { marketCap: 0, dailyVolume: 0 },
-  } = useGetCoingeckoCoin();
+    data: { market_cap, volume_24h, circulating_supply, total_supply } = {
```

Comment:
Normalize circulating_supply -> circulatingSupply in usePrice hook

Reply:
I think we can follow API name fields, no need to map the name again which make code more complicated